languages/emacs-lisp/reference/README.md

```diff
@@ -0,0 +1,39 @@
+# Emacs Lisp reference
+
+## Concepts
+
+The Emacs Lisp concept exercises are based on concepts.
+The list below contains the concepts that have been identified for the Emacs Lisp language.
+
+See [progression.md](progression.md) for planning the progression order of the Emacs Lisp track.
+
+The core features a Emacs Lisp developer should know about are:
+
+- Special forms
+- Atoms
+- Quote
+- S-expressions
+- [Evaluation][evaluation]
+- [Functions][functions]
+- [Variables][variables]
+- [Macros][macros]
+- Side-effects
```

Comment:
what do you mean here?

Reply:
About side effects? The example given in the intro doc is the `set` function.

Reply:
It looks close to the concept of `variable` itself.

```This is the concept of assigning a name to a piece of data.``` (from `reference/concepts`)